services/supervisor/src/schema.rs

```diff
@@ -68,9 +68,9 @@ impl<'a, T: IndexAccess> Schema<'a, T> {
         )
     }
 
-    pub fn pending_proposal(&self) -> Entry<T, ConfigProposalWithHash> {
-        Entry::new(
-            [self.instance_name, ".pending_proposal"].concat(),
+    pub fn pending_propose_hashes(&self) -> ProofMapIndex<T, u64, ConfigProposalWithHash> {
```

Comment:
Maybe implement `BinaryKey` and `ObjectHash` for `Height` and use this type for key instead of `u64`?